app/code/Magento/Review/Test/Unit/Observer/CatalogProductListCollectionAppendSummaryFieldsObserverTest.php

```diff
@@ -0,0 +1,138 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types = 1);
+
+namespace Magento\Review\Test\Unit\Observer;
+
+use Magento\CatalogSearch\Model\ResourceModel\Fulltext\Collection;
+use Magento\Framework\Event;
+use Magento\Framework\Event\Observer;
+use Magento\Review\Model\ResourceModel\Review\Summary;
+use Magento\Review\Model\ResourceModel\Review\SummaryFactory;
+use Magento\Review\Observer\CatalogProductListCollectionAppendSummaryFieldsObserver;
+use Magento\Store\Api\Data\StoreInterface;
+use Magento\Store\Model\StoreManagerInterface;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Test class for \Magento\Review\Observer\CatalogProductListCollectionAppendSummaryFieldsObserver
+ */
+class CatalogProductListCollectionAppendSummaryFieldsObserverTest extends TestCase
+{
+    /**
+     * @var Event|\PHPUnit_Framework_MockObject_MockObject
```

Comment:
Actually with PHPUnit 6 you can use `\PHPUnit\Framework\MockObject\MockObject`

Reply:
**done**